packages/amplify-provider-awscloudformation/lib/build-resources.js

```diff
@@ -84,6 +90,20 @@ function getSourceFiles(dir, ignoredDir) {
   }, []);
 }
 
+function removeOutdatedPackage(currentBuildFile) {
+  try {
+    const distDir = path.dirname(currentBuildFile);
+    const deletePromises = fs
+      .readdirSync(distDir)
+      .map(p => path.join(distDir, p))
+      .filter(p => currentBuildFile !== p)
+      .map(p => fsExtra.remove(p));
+    return Promise.all(deletePromises);
+  } catch (e) {
+    // nothing to do here
```

Comment:
Should probably show some error log out here? although not neccessarily throw it?

Reply:
I don’t think the error message is useful as there is nothing user can do about it. Also not deleting doesn’t cause any problem except using disk space

Reply:
The user can manually delete it then maybe? It would also help us know what the issue is (if there's any). If we don't show anything, we would just silently fail and never know if someothing is going wrong